index.js

```diff
@@ -0,0 +1,103 @@
+//задачи на основы js
+alert("I JavaScript!");
+
+//задачи на переменные js
+//task1
+let admin
+, name;
+
+name = "John";
+
+admin = name;
+
+alert(admin);
+
+//task2
+let planetName = "Earth";
+
+let personalUserName = "Dasha";
+
+//задача на типы js
+//что выведет этот скрипт
+let name = "Ilya";
+
+alert( `hello ${1}` ); // выведет Hello Ilya
```

Comment:
```suggestion
alert(`hello ${1}`); // выведет Hello Ilya
```
пробелы в скобках не нужны